### PR TITLE
[159] Create & Style the Customer Signup Page using CSS -Frontend

### PR DESCRIPTION
**Pull request by Immaculeta1**
created and styled signup page

### CODE REVIEW

**Comment by Immaculeta1** [comment]
Capture1
Capture2

File: submissions/nwokejeme-immaculeta/html-forms/homepage.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    <section class="container">
+        <main><br> <br>
+            <h1>Register your Account</h1> 
+            <p>Registering your account gives you access to sweet discount, promo and events available only to registered customers. </p> <br> <br>
+           <p>Already got an account? <a href="#">Sign in</a></p>
+        </main>
+        <aside>
+            <form id=validateForm action="#">
+                <div class="form-details">
+                    <label for="firstName">First Name</label>
```
Comment (ToluAkin) [inline submissions/nwokejeme-immaculeta/html-forms/homepage.html:34]: @Immaculeta1 Remove all `br` elemenets in this file and style it with css

File: submissions/nwokejeme-immaculeta/html-forms/homepage.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                    <br>
+                    <input type="text" name="firstName" id="firstName" maxlength="50">
```
Comment (ToluAkin) [inline submissions/nwokejeme-immaculeta/html-forms/homepage.html:36]: There is no need to add text limit to your input field. That should come in when validating the form @Immaculeta1

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/nwokejeme-immaculeta/html-forms/homepage.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
+<html>
```
Comment (adaugochi) [inline submissions/nwokejeme-immaculeta/html-forms/homepage.html:1]: Why is this file called `homepage.html`? thought this is signup page. 

Same applies to the css file name

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Comment by adaugochi** [comment]
Send a screenshot of the mobile view of what you dis

**Comment by Immaculeta1** [comment]
Capture2
s.githubusercontent.com/74491329/103157336-19f26980-47b2-11eb-8db5-5cd0408cda5b.PNG)
Capture1

File: submissions/nwokejeme-immaculeta/html-forms/signup.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                <li>
+                    <a href="#"><img src="../HTML/image/youtube.png" alt="Youtube"></a>
+                </li>
+            </ul>
+        </div>
+        <br>
+        <p>Copyright &copy; 2020, Taries Empire All rights reserved</p>
+    </footer>
+
+    <script src="../domManipulation/signUp.js"></script>
```
Comment (adaugochi) [inline submissions/nwokejeme-immaculeta/html-forms/signup.html:104]: Have you gotten to javaScript yet? How come you have these js files

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Comment by adaugochi** [comment]
Well done @Immaculeta1 

Kindly rework your newsletter email input field to look more better
image

**Pull request merged**